doc/operator-upgrade.md

```diff
@@ -82,18 +82,26 @@ For upgrading Operator to version 0.5.0, the following must be true:
 See: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion
 
 2. Cert-Manager or some other certificate management solution must be deployed for managing webhook service certificates. The upgrade trigger script assume user has [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) installed but any other cert management solution can also be used and script would need to be modified accordingly.
-
-  Installing cert-manager: https://cert-manager.io/docs/installation/kubernetes/
+Installing cert-manager: https://cert-manager.io/docs/installation/kubernetes/
 
 3. [Bookkeeper Operator](https://github.com/pravega/bookkeeper-operator/tree/master/charts/bookkeeper-operator) must be deployed in the same namespace as Pravega Operator, prior to triggering the upgrade.Also, Bookkeeper operator config map should contain the bookkeeper versions of the installed bookkeeper.
 
-4. Additionally, modify the version map present at this location (https://github.com/pravega/pravega-operator/blob/master/tools/manifest_files/version_map.yaml), to include the pravega version installed.
+4. Install an Issuer and a Certificate (either self-signed or CA signed) in the same namespace as the Pravega Operator (refer to [this](https://github.com/pravega/pravega-operator/blob/master/deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace).
+> The name of the certificate (*webhookCert.certName*), the name of the secret created by this certificate (*webhookCert.secretName*), the tls.crt (*webhookCert.crt*) and tls.key (*webhookCert.key*) need to be specified against the corresponding fields in the values.yaml file, or can be provided with the upgrade command as shown [here](#triggering-the-upgrade).
+The values *tls.crt* and *tls.key* are contained in the secret which is created by the certificate and can be obtained using the following command
+```
+kubectl get secret <secret-name> -o yaml | grep tls.
+```
+
+5. Execute the script `pre-upgrade.sh` inside the `scripts` folder. This script patches the `pravega-webhook-svc` with the required annotations and labels.
 
 
```

Comment:
We have the upgrade script present in the tools folder a better option, in my opinion, would be to not remove the documentation part on how to use it rather we can just say a better or recommended way of upgrading operator from 0.4.x to 0.5.0 is using helm and mention the steps for it.

Reply:
The script still exists but we should not be mentioning both the approaches in the same document is what I feel. Hence even though the script is still there we should still mention just the helm approach here. Both approaches mentioned in the same place would just increase confusion.